frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -121,14 +121,6 @@ export const VMDetailsList: React.FC<VMResourceListProps> = ({
         <VMStatuses vm={vm} vmi={vmi} pods={pods} migrations={migrations} />
       </VMDetailsItem>
 
```

Comment:
:-) things change ... the reason is that the vm and vmi page show the same data ... so no new info ...
p.s.
leaving it will require the page to know if it's a vm of vmi page ...

Reply:
I see